docs/content/manual/release-specific/v1.1.1/tolerations_setting.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Set Tolerations For System Components and User-deployed
+Components Separately
+---
+
+## Related issue
+https://github.com/longhorn/longhorn/issues/2120
+
+Manual Tests:
+
+### Case 1: Existing Longhorn installation
+1. Install Longhorn master.
+1. Change toleration in UI setting
+1. Verify that `longhorn.io/last-applied-tolerations` annotation and `toleration` of manager, drive deployer, UI are not changed.
+1. Very that `longhorn.io/last-applied-tolerations` annotation and `toleration` for managed components (CSI components, IM pods, share manager pod, EI daemonset) are updated correctly
+
+### Case 2: New installation by Helm
+1. Install Longhorn master, set tolerations like:
+  ```
+  defaultSettings:
+    taintToleration: "key=value:NoSchedule"
+
+  longhornManager:
+    priorityClass: ~
+    tolerations:
+    # If you want to set tolerations for Longhorn Manager DaemonSet, delete the `[]` in the line above
+    # and uncomment this example block
+    - key: key
+      operator: Equal
+      value: value
+      effect: NoSchedule
+
+  longhornDriver:
+    priorityClass: ~
+    tolerations:
+    # If you want to set tolerations for Longhorn Manager DaemonSet, delete the `[]` in the line above
+    # and uncomment this example block
+    - key: key
+      operator: Equal
+      value: value
+      effect: NoSchedule
+
+  longhornUI:
+    priorityClass: ~
+    tolerations:
+    # If you want to set tolerations for Longhorn Manager DaemonSet, delete the `[]` in the line above
+    # and uncomment this example block
+    - key: key
+      operator: Equal
+      value: value
+      effect: NoSchedule   
+  ```
+3. Verify that the toleration is added for: IM pods, Share Manager pods, CSI deployments, CSI daemonset, the backup jobs, manager, drive deployer, UI
+4. Uninstall the Helm release. Verify that the uninstallation success
```

Comment:
If I have taints added for the node where I want to run the uninstaller job, the tolerations should be added to that uninstaller yaml file too, isn't it?

Reply:
The uninstalling job will have the same toleration as Longhorn manager ds if users install by Helm and set toleration for manager ds.